esp-wifi/src/wifi/mod.rs

```diff
@@ -499,6 +650,7 @@ pub mod ipv4 {
     impl FromStr for Mask {
         type Err = &'static str;
 
+        /// Parses a string to create a `Mask`.
```

Comment:
I don't think this or any other trait impl needs to be documented. The traits are pretty obvious and have their own docs. There are quite a few of these.